src/js/components/DataView/__tests__/__snapshots__/DataView-test.tsx.snap

```diff
@@ -68,8 +68,8 @@ exports[`DataView preset view 1`] = `
   display: flex;
   box-sizing: border-box;
   max-width: 100%;
-  margin-left: 12px;
-  margin-right: 12px;
+  margin-left: 16px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below